src/SFA.DAS.EAS.EmployerAccount.Events.UnitTests/MessageTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Events.Messages;
+
+namespace SFA.DAS.EAS.EmployerAccount.Events.UnitTests
+{
+    public class MessageTests
+    {
+        private const string SignedByName = "signedByName";
+        private const string TestStringProperty = "TestStringProperty";
+        private const long AccountId = 12345;
+        private const long TestLongProperty = 54321;
+
+        [Test]
+        public void WhenAMessageIsSerializedThenItCanBeDesialized()
+        {
+            var testClass = new TestMessage(SignedByName, AccountId, TestStringProperty, TestLongProperty);
+            var serialized = JsonConvert.SerializeObject(testClass);
+            var deserialized = JsonConvert.DeserializeObject<TestMessage>(serialized);
```

Comment:
If message is going to be serialisable we need to give it the correct serialisable interface as well.

Reply:
You mean Iserializable? All of our nessages will be serialised and deserialised. The reason for this test is to make sure the json serializer can do it's stuff when I've removed the setters.

Reply:
Yep ISerialiable, The tests are fine I just placed it there as a note as this was the place the serialisation was mentioned. Originally we did specifically state they were so now we are its worth just crossing the i's and dotting the T's. The tests are fine :)